### PR TITLE
Move ARFG admin content to Training Center

### PR DESCRIPTION
**Pull request by lemonez**
[DOCS-160](https://algorithmia.atlassian.net/browse/DOCS-160) - ARFG admin code docs

Didn't update screenshots in dev center but did update in training center.

## Checklist
- [x] My PR title includes a relevant Jira ticket name
- [ ] If no associated Jira ticket, my PR title is informative (individual commit messages are squashed in this repo)
- [x] Unnecessary text (notes, TODOs, etc.) has been removed
- [x] For YAML "front matter":
  - Properties are alphabetized
  - `permalink` and `redirect_from/to` properties have `/` at beginning and end (e.g., `/path/`)
- [ ] Screenshots, if present, follow the [Screenshot guidelines](https://algorithmia.atlassian.net/wiki/spaces/CUSTOMERS/pages/1634861478/CFD+Style+Guide#Screenshots) which means:
  - Full browser width (browser at 100% zoom and 10-12" width)
  - Max 1000px
- [x] Grammar, style, tone, punctuation, etc. follow the Algorithmia [manual of style](https://docs.google.com/document/d/1PPVfgMkX7-EVGLPMhN1E485CAXu9QfhSLKM0lZhnZdU/edit?usp=sharing)
- [x] Headers are “Sentence case with Proper Nouns capitalized”
- [x] Placeholder strings are `UPPER_SNAKE_CASE` and are named as in the [Glossary](https://docs.google.com/document/d/1bYs0j_a4v8LbkbS8r0x2SZ_J6mZ4sIt4w_60dq8iGh4/edit#heading=h.qov2yks2a685) where appropriate

### CODE REVIEW

**Comment by lemonez** [comment]
Ok I hadn't completely understood the feature until I got hands on. I added a few more sections and updated images accordingly so they should all now conform to the screenshot guidelines.

**Review by warmbowski — DISMISSED** [review]
lgtm 🍾

File: _pages/platform/advanced-governance-reporting.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+![Metrics drop-down](/developers/images/post_images/advanced_reporting/governance_reporting_metrics_list.png)
+
+## Permissions
+
+The governance reporting feature is available to cluster admins and organization admins. In order for cost information to be estimated, a cluster admin must first [set expected resource cost rates](https://training.algorithmia.com/exploring-the-admin-panel/836479#setting-cost-rates). If the cost rates haven't been set, all estimated costs will default to 0.
+
+### Cluster admin
+
+Cluster admins can view all resource consumption data captured within the platform, grouped by organization, account, and algorithm. For more details, see the [cluster admin docs](https://training.algorithmia.com/exploring-the-admin-panel/836479).
```
Comment (warmbowski) [inline _pages/platform/advanced-governance-reporting.md]: Should this be added, something like "...as well as everything an Org Admin can view"?

Comment (lemonez) [reply _pages/platform/advanced-governance-reporting.md]: thanks. added.

File: _pages/release-notes.md [hunk 2]
```diff
@@ -10,6 +10,30 @@ title:  "Release Notes"
 tags: [alg-dev-getting-started, app-dev-getting-started]
 ---
 
+## Advanced Reporting for Governance
+
+The Advanced Reporting for Governance feature provides administrators the ability to view resource usage on the platform. Usage can be viewed at four different levels depending on your access permissions. If you’re an organization administrator, you can explore reporting at the organization and account level, and even down to the individual algorithm level. Cluster administrators are granted all of the above permissions, as well as being able to view reporting at the cluster level. We’ll start at the top level of the cluster roll-up.
```
Comment (warmbowski) [inline _pages/release-notes.md:15]: Cluster admins are the only ones that can add the base cost information. Not sure if you need this here.

**Review by warmbowski — DISMISSED** [review]
One comment. Looks good

File: _pages/release-notes.md (hunk 2 above)
Comment (lemonez) [reply _pages/release-notes.md:15]: yeah i think that's probably good to mention. added in next commit

**Pull request merged**